### PR TITLE
ptbr translation

### PR DESCRIPTION
**Pull request by mezysinc**

### CODE REVIEW

**Comment by fm-sys** [comment]
Thanks for providing this PR, very welcome!

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <string name="app_update_summary">As atualizações contêm correções de falhas ou novas funcionalidades. A instalação é recomendada…</string>
+    <string name="app_update_install">Instalar</string>
+    <string name="app_update_show_details">Mais detalhes</string>
+    <string name="app_update_download_info">Baixando atualização</string>
+    <string name="app_update_download_text">Deve levar apenas alguns segundos…</string>
+    <string name="download_successful">Foi baixado com sucesso</string>
+    <string name="notification_channel_name">Download concluído</string>
+    <string name="title_activity_about">Sobre o Snapdrop para Android</string>
+    <string name="title_activity_about_short">Sobre o Snapdrop</string>
+    <string name="developer_notice" translatable="false">Snapdrop para Android por <a href="https://github.com/fm-sys">fm-sys</a></string>
```
Comment (fm-sys) [inline app/src/main/res/values-pt-rBR/strings.xml:14]: You are totally right, that this string should also be translated. Could you therefore please remove the `translatable="false"` flag from this string resource. Best both here and in `values/strings.xml`? 

Thanks for helping to translate this app!

Comment (mezysinc) [reply app/src/main/res/values-pt-rBR/strings.xml]: Hello fm-sys, 
I removed this section from both values. Thanks

**Comment by fm-sys** [comment]
@mezysinc thanks for this contribution! ❤️

**Pull request merged**